automation/build-copr.sh

```diff
@@ -16,15 +16,19 @@ cleanup_artifacts() {
 mvn_write_settings() {
```

Comment:
We add the snapshot repository already in the default pom. Is there a reason we need it here also? Think it can just be dropped?